<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseColor</class>
 <widget class="QWidget" name="ChooseColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="slider_red">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_green">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_blue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_done">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_random">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
